src/shared/services/jwt/jwt.service.ts

```diff
@@ -22,6 +23,24 @@ const createToken = async (
   });
 };
 
+export const checkRefreshToken = async (
+  jwtSecret: string,
+  token: string
+): Promise<DecodeTokenOutput> => {
+  const cleanedUpCode = token.replace('Bearer ', '');
+
+  const tokenData = await JWTService.decodeToken(cleanedUpCode, jwtSecret);
+
+  if (tokenData && tokenData.exp) {
```

Comment:
You don't need to check `exp` - it should be done automatically during the token verification by the lib. Please check if it's true